src/Api.elm

```diff
@@ -131,6 +144,14 @@ sessionTrackIdDecoder =
         ]
 
 
+sessionLocationIdDecoder : Json.Decode.Decoder (Maybe LocationId)
+sessionLocationIdDecoder =
+    Json.Decode.oneOf
```

Comment:
I believe you can use `Json.Decode.Maybe Json.Decode.int` to make this a one liner?